huaweicloud/services/dataarts/common.go

```diff
@@ -18,6 +18,7 @@ import (
 // + Data Service:
 //   - DLM.4205: catalog does not found.
 //   - DLM.3027: Permission set does not found.
+//   - DLS.1000: Data secrecy level does not found.
```

Comment:
This error code is not belong to `Data Service`, should be `Security`

Reply:
fixed